server/domain/academic-record/models/AcademicRecord.ts

```
@@ -0,0 +1,182 @@
+      totalEarnedCredits: number | null;
+      cumulativeGpa: number | null;
+      percentile: number | null;
+      attemptedCreditsGpa: number | null;
+    };
+    enrollments: {
+      offeringId: number;
+      grade?: GradeType;
+      points: number;
+      isRetake: boolean;
+      originalScore?: number;
+    }[];
```

ESLint

[error] Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.

(@typescript-eslint/array-type)